Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -532,16 +542,19 @@
 }
 
 void CGContextCairo::CGContextSetTextMatrix(CGAffineTransform matrix) {
-    curState->curTextMatrix = matrix;
+    curState->curTextMatrix.a = matrix.a;
+    curState->curTextMatrix.b = matrix.b;
+    curState->curTextMatrix.c = matrix.c;
+    curState->curTextMatrix.d = matrix.d;
```

Comment:
Can you add tests for this into the standard CoreGraphics tests?

Here's what I'm seeing:
```
CGContextSetTextMatrix(ctx, CGAffineTransformMake(1, 0, 0, 1, 10, 3))
CGContextGetTextMatrix(ctx)
{a:1,b:0,c:0,d:1,tx:10,ty:3}

CGContextGetTextPosition(ctx)
{x:10,y:3}

CGContextSetTextPosition(ctx, 100, 300)
CGContextGetTextPosition(ctx)
{x:100,y:300}

CGContextGetTextMatrix(ctx)
{a:1,b:0,c:0,d:1,tx:100,ty:300}
```